package.json

```
@@ -44,7 +44,8 @@
     "serialize-javascript": "^3.1.0",
     "**/**/@types/node": "^12.0.0",
     "**/**/minimist": "^1.2.0",
-    "**/mkdirp": "^0.5.1"
+    "**/mkdirp": "^0.5.1",
+    "keytar": "7.7.0"
```

```suggestion
    "keytar": "^7.7.0"
```

we can accept more recent version as well (but let check first how it goes with that :-)